pwiz_tools/Skyline/Skyline.csproj

```diff
@@ -3324,6 +3325,7 @@
       <Link>unimod.obo</Link>
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </Content>
+    <None Include="Method\Shimadzu\BuildShimadzuMethod.exe.config" />

```

Comment:
This should be changed so that the ".exe.config" gets copied to the output directory and included in the installers.
Skyline now ships with all of the .exe.config files  since they tell Windows which .Net version the exe needs. If the .exe.config is missing, Skyline forces the user to install .Net 3.5.